shared/src/main/java/gov/hhs/cdc/trustedintermediary/context/Environment.java

```diff
@@ -0,0 +1,21 @@
+package gov.hhs.cdc.trustedintermediary.context;
+
+public class Environment {
+
+    private Environment() {}
+
+    public static String getProperty(String key) {
+        if (DotEnv.isNull()) {
```

Comment:
Will DotEnv ever be null? It's getting initialized in the DotEnv class on instantiation

Reply:
Good point. The .env file may not exist, but in that case the library will throw an exception instead of returning null, so we need to handle that

Reply:
We opted to not continue with the Environment class as the Dotenv library is doing some of that functionality behind the scenes.